demo/src/screens/MainScreen.js

```diff
@@ -72,6 +99,7 @@ class MainScreen extends Component {
     const data = this.getMenuData();
 
     if (buttonId === 'uilib.settingsButton') {
+      console.log(`2nd call to pushScreen: ${this.settingsScreenName}`);
```

Comment:
All the console.logs here are not mine, all are related to the search bar and his render functions.
Should I delete all of them?
I guess It marked up as changed because of some spacing when reformat the file.